Trippy/Utils/ParseUtils.h

```diff
@@ -9,18 +9,23 @@
 #import "Parse/Parse.h"
 #import "LocationCollection.h"
 #import "Location.h"
+#import "Itinerary.h"
```

Comment:
Good example of a previous point here - you can forward declare the `Itinerary` class instead of adding the header here. Then, any .m files that need to know what the `Itinerary` class actually is can import the header.